v21.1/show-backup.md

```diff
@@ -301,6 +302,57 @@ Or, use the `kms` option and the same KMS URI that was used to create the backup
 (20 rows)
 ~~~
 
+### Show a backup's internal metadata
+
+Use the `WITH as_json` option to output a backup's internal metadata, contained in its manifest file, as a JSON value:
+
+{% include copy-clipboard.html %}
+~~~ sql
+SHOW BACKUP '/2021/11/15-150703.21' IN 's3://test/backup-test?AWS_ACCESS_KEY_ID=[placeholder]&AWS_SECRET_ACCESS_KEY=[placeholder]' WITH as_json;
+~~~
+
+The response will include a `manifest` column with the file's contents as the JSON value. Use [JSONB functions](functions-and-operators.html#jsonb-functions) to query particular data or edit the format of the response.
+
+{{site.data.alerts.callout_info}}
+The response returned from `SHOW BACKUP ... WITH as_json` is a backup's internal metadata. This content is subject to change from version to version of CockroachDB and does not offer the same stability guarantees as the other `SHOW BACKUP` [options](#options) and their [responses](#response). As a result, `as_json` should **only** be used for debugging or general inspection purposes.
+{{site.data.alerts.end}}
+
+For example, to return a specific entry from the JSON response as a [`string`](string.html) indented and with newlines use the [`jsonb_pretty()`](functions-and-operators.html#jsonb-functions) function:
+
+{% include copy-clipboard.html %}
+~~~ sql
+SELECT jsonb_pretty(manifest->'entryCounts') AS f FROM [SHOW BACKUP '/2021/11/15-150703.21' IN 's3://test/backup-test?AWS_ACCESS_KEY_ID=[placeholder]&AWS_SECRET_ACCESS_KEY=[placeholder]' with as_json];
+~~~
+
+~~~ json
+  {
+      "dataSize": "458371",
+      "indexEntries": "1015",
+      "rows": "2565"
+  }
+~~~
+
+To query for particular data, use the `jsonb_array_elements()` function to expand the desired elements from the JSON response. The following query returns the paths to each of the data files within the backup:
```

Comment:
it would be cool if this JSONB function also linked to functions-and-operators.html#jsonb-functions